apps/extension/src/utils/index.ts

```diff
@@ -185,12 +185,13 @@ export const toPublicAccount = (derivedAccount: DerivedAccount): Account => {
     alias,
     address,
     type,
-    pseudoExtendedKey,
     source,
```

Comment:
One last comment - if we don't need `source` on transparent, I would maybe omit it here. It gets appended below for shielded which makes sense!

Reply:
Yeah I forgot to remove the line :D

Reply:
done!